gradle_support/linters.gradle

```diff
@@ -1,7 +1,6 @@
-import config.Plugins
```

Comment:
You can convert it into plugin via included build, as you've done for dependency updater